packages/atchops/include/atchops/rsa.h

```diff
@@ -16,8 +22,8 @@
  * @param signatureolen the length of the signature buffer after signing
  * @return int 0 on success
  */
-int atchops_rsa_sign(atchops_rsakey_privatekey privatekey, mbedtls_md_type_t mdtype, const unsigned char *message,
-                     const unsigned long messagelen, unsigned char *signaturebase64,
+int atchops_rsa_sign(const atchops_rsakey_privatekey privatekey, const mbedtls_md_type_t mdtype,
```

Comment:
Should we use a const pointer instead of just a const object to avoid making a copy? 

`const atchops_rsakey_privatekey *privatekey`

atchops_rsakey_privatekey is a humungous struct

Reply:
This can be changed when someone goes to work on it.